packages/tiled/lib/src/common/color.dart

```diff
@@ -0,0 +1,25 @@
+part of tiled;
+
+/// Basic data class holding a Color in ARGB format.
+/// This can be converted to dart:ui's Color using the flame_tiled package
+class ColorData {
+  static int _sub(int hex, int index) => (hex >> index * 8) & 0x000000ff;
+
+  final int red;
+  final int green;
+  final int blue;
+  final int alpha;
+
+  /// Parses the Color from an int using the lower 32-bits and tiled's format: 0xaarrggbb
+  ColorData.hex(int hex)
+      : alpha = _sub(hex, 3),
+        red = _sub(hex, 2),
+        green = _sub(hex, 1),
+        blue = _sub(hex, 0);
+
+  const ColorData.rgb(this.red, this.green, this.blue, [this.alpha = 255])
```

Comment:
"rgba"? and why deviate from "argb"?

Reply:
I don’t really know, it just felt more natural. For most colors the alpha value is just 0xff so it can have a default value at the end. CSS also puts it at the end #rrggbbaa, I believe for the same reason. 

This constructor is meant for users and not used internally, so I think this way is a better experience. However I‘m happy to change it!

Reply:
Right, I was trying to stick with the dart:ui color for familiarity.